php/src/Communicator.cpp

```diff
@@ -476,7 +476,7 @@ ZEND_METHOD(Ice_Communicator, getProperties)
     try
     {
         Ice::PropertiesPtr props = _this->getCommunicator()->getProperties();
-        if (!createProperties(return_value, props))
+        if (!IcePHP::createProperties(return_value, props))
```

Comment:
Without the qualification, the new Ice::createProperties template function is found first via ADL.